invenio_app_ils/ui/src/invenio_app_ils/forms/core/SelectField.js

```diff
@@ -28,6 +29,7 @@ export class SelectField extends Component {
     const {
       form: { values, setFieldValue, handleBlur, errors },
     } = props;
+    const defaultValue = getIn(this.props, 'options.0.value', '');
```

Comment:
what will happen if for some reason there will be no options provided?

Reply:
If no options are provided `Form.Dropdown` will complain about missing `options` prop and the `defaultValue` will be set to `''`.